Modernise several formula

afh opened the pull request.

tdsmith commented:
`diction` test and audit are failing; the test failure is probably because the decimal separator is `.` in the test bot's locale. That particular test should be more concise; I'm probably happy if `echo This is a test of GNU diction. | style` (or its moral equivalent) produces output including `7 words`.

afh commented:
@tdsmith Thanks for you feedback, I've added `ENV["LC_ALL"] = "C"` to the test and updated the expected output accordingly.

I find having a more extensive test as an example helpful in roughly judging what the software does.

mikemcquaid commented on Library/Formula/diction.rb:
```diff
@@ -1,14 +1,68 @@
+    violation. Unless he is certain of doing as well, he will probably do best
+    to follow the rules. After he has learned, by their guidance, to write
+    plain English adequate for everyday uses, let him look, for the secrets of
+    style, to the study of the masters of literature.
```
Ident this one further.

mikemcquaid commented on Library/Formula/diction.rb:
```diff
@@ -1,14 +1,68 @@
+
+#{file}:5: After he has learned, by their guidance, to write plain English [adequate] for everyday uses, let him look, for the secrets of style, to the study of the masters of literature.
+
+4 phrases in 4 sentences found.
```
Ident this fully.

mikemcquaid commented on Library/Formula/diction.rb:
```diff
@@ -1,14 +1,68 @@
+
+4 phrases in 4 sentences found.
+    EOS
+    assert_equal diction_result, shell_output("#{bin}/diction #{file}")
```
I don't think we need to match the contents exactly for two comments; it'll be a bit error prone. I think checking exit codes and/or partial output will be sufficient.

mikemcquaid commented on Library/Formula/ktoblzcheck.rb:
```diff
@@ -16,6 +14,16 @@ def install
+    Bank: 'Bundesbank' at 'Berlin' (10000000)
+    Account: 123456789
+    Result is: (0) Ok
+
```
Ident this a line further.

mikemcquaid commented on Library/Formula/ktoblzcheck.rb:
```diff
@@ -16,6 +14,16 @@ def install
+    Result is: (0) Ok
+
+    EOS
+    assert_equal result, shell_output("#{bin}/ktoblzcheck 10000000 123456789")
```
Again, I think matching exact string output here will be potentially error-prone. Partial output and/or exit code checking seems enough.

mikemcquaid commented on Library/Formula/lout.rb:
```diff
@@ -1,20 +1,47 @@
+    delineation of her characters, of the subtlety of psychological analysis and
+    the maturity of moral comment which underlie this complex and varied novel of
+    English provincial life in the early nineteenth century.
+    @End @Text
```
Ident this further.

mikemcquaid replied:
And don't need this to be as long as it is, I think.

mikemcquaid commented on Library/Formula/lout.rb:
```diff
@@ -1,20 +1,47 @@
+          century.
+\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n
+    EOS
+    assert_equal result, shell_output("#{bin}/lout -p #{input}")
```
Same as above; partial output and exit codes.

mikemcquaid commented on Library/Formula/objc-run.rb:
```diff
@@ -1,14 +1,12 @@
-    bin.install 'objc-run'
-    (share+'objc-run').install 'examples', 'test.bash'
+    bin.install "objc-run"
+    (share+"objc-run").install "examples", "test.bash"
```
Probably don't want to have any formulae with just string changes.

mikemcquaid commented on Library/Formula/qrencode.rb:
```diff
@@ -1,5 +1,3 @@
-require "formula"
-
```
Similarly, probably don't want to change this as it'll show in `brew update` currently.
